test_runner/regress/test_import.py

```diff
@@ -263,6 +263,7 @@ def _import(
     # Import using another tenant_id, because we use the same pageserver.
     # TODO Create another pageserver to make test more realistic.
     tenant = TenantId.generate()
+    timeline = TimelineId.generate()
```

Comment:
This overrides the timeline ID passed as argument. Looks wrong

Reply:
It was source timelin-id. I wonder if it is correct to have two different tenants with the same timeline-id?
Yes, I know that only pair tenant/timeline is expected to be unique.
But is there any sense to preserve timeline-iod during import?

Reply:
Ah, I see. Ok yeah it's a bit weird, but it should work.

In any case, that's not related to persisting pgstat, so let's create a separate PR for that